src/components/SingleItem.js

```diff
@@ -83,9 +84,20 @@ const SingleItem = (props) => {
   };
 
   const handleDelete = async (e) => {
-    if (window.confirm(`Do you really want to delete ${name}?`)) {
-      await firestore.collection('items').doc(id).delete();
-    }
+    swal({
+      title: 'Are you sure?',
+      text: `Do you really want to delete ${name}?`,
+      icon: 'warning',
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
+      if (willDelete) {
+        firestore.collection('items').doc(id).delete();
+        swal(`You have deleted ${name}.`, {
+          icon: 'success',
+        });
+      }
+    });
```

Comment:
I love the implementation of this! I haven't used SweetAlert before, but I feel like I can understand what's going on here because the code looks clean and simple. 🚀